frontend/src/features/comparisons/CriteriaSlider.tsx

```diff
@@ -100,42 +98,22 @@ const CriteriaSlider = ({
         </Grid>
       </div>
       <div className={classes.sliderContainer}>
-        <IconButton
-          aria-label="left"
-          onClick={() => handleSliderChange(criteria, -10)}
-          style={{
-            color: 'black',
-            transform: 'rotate(180deg)',
-            padding: 0,
-          }}
-          disabled={disabled}
-        >
-          <DoubleArrowIcon />
-        </IconButton>
         <Slider
-          // ValueLabelComponent={ValueLabelComponent}
           id={`slider_expert_${criteria}`}
           aria-label="custom thumb label"
```

Comment:
In mui version 5 the sliders are bigger by default. I would personally prefer with the option size="small".

![Capture d’écran de 2021-12-30 05-44-03](https://user-images.githubusercontent.com/47578089/147723220-24ffc1bb-a718-4c8d-8042-d7349c0fe15a.png)

Reply:
I found that it looks nice with the bigger sliders. What are your opinions @aidanjungo & @lenhoanglnh ?

Reply:
I personally like the new style, it makes the discrete values more visible.

Reply:
The markers are an improvement, it is mainly the blue circles that I find a little too big. But maybe everyone else like it.